include/psa/crypto_values.h

```diff
@@ -279,6 +279,11 @@
  * to read from a resource. */
 #define PSA_ERROR_INSUFFICIENT_DATA     ((psa_status_t)-143)
 
+/** This can be returned if a function can no longer operate correctly.
+ * For example, if an essential initialization operation failed or
+ * a mutex operation failed. */
+#define PSA_ERROR_SERVICE_FAILURE       ((psa_status_t)-144)
+
```

Comment:
Remark (no changes requested): Technically we don't need this just yet, but I guess there is no harm in adding it here.

Reply:
I put this in here because the 3 following PRs for the next issues in this epic all need this, and those prs should ideally be independent of each other. If we do go ahead with adding macros for locking and unlocking this could be added then, but as you said it doesn't do any harm to add it here.

Reply:
Good thinking!